src/osd/scrubber_common.h

```diff
@@ -17,6 +17,25 @@ namespace Scrub {
 /// high/low OP priority
 enum class scrub_prio_t : bool { low_priority = false, high_priority = true };
 
+// possible outcome when trying to select a PG and scrub it
+enum class attempt_t {
+	scrubbing,  // successfully started a scrub
```

Comment:
i'd suggest use two spaces instead of a tab (8 spaces) for indent here.

Reply:
OK. I'll try to configure clang-format for that.
(I am trying to avoid hand-made formatting)